src/PkgAuthentication.jl

```diff
@@ -194,61 +218,66 @@ function step(state::ClaimToken)::Union{ClaimToken, HasNewToken, Failure}
 
         if haskey(body, "token")
             return HasNewToken(state.server, body["token"])
-        elseif haskey(body, "expiry")
+        elseif haskey(body, "expiry") # QUESTION: "expiry" or "expires"?
```

Comment:
`expiry` is [correct](https://github.com/JuliaComputing/JuliaTeam/blob/de0f30f4663e997b9ac60ce91914f44d5f7500ba/teamsrvr/common/lua/browserauth.lua#L217).

Reply:
Not your issue since its server-side, but it seems like what the server is serving here is a bit of a mess. We should make what the server actually serves match what the client currently expects.

Reply:
This `expiry` response is different from `expires_in`/`expires_at` in auth.toml.

A given challenge/response pair is only valid for 120s, during which the user must log in in their browser.

Reply:
I guess I'm not getting the `expiry` thing. What does this field signify?

Reply:
Ok, so the auth flow is this:

1. Client sends `challenge` to server.
2. Server responds to `challenge` with `response` and saves those plus an expiry 120s in the future in the DB.
3. Client opens browser at `/response?$(response)`. When browser auth is successful, the server "activates" the `challenge`/`response` pair.
4. Client polls `/claim_token` with the `challenge`/`response` pair. This can result in a bunch of different answers:
4.1. If the `challenge`/`response` pair is active and not expired, send back the token. 
4.2. If the `challenge`/`response` pair is not active, send back when it expires (this is the `expiry` field). 
4.3. If the `challenge`/`response` is expired or not present in DB, send back an error.

So the client needs to stop polling once the `challenge`/`response` pair expires.